Makefile.include

```diff
@@ -216,6 +216,14 @@ include $(RIOTMAKE)/defaultmodules.inc.mk
 include $(RIOTBOARD)/$(BOARD)/Makefile.include
 include $(RIOTCPU)/$(CPU)/Makefile.include
 
+# Add vendor header includes if present
+CPUVENDOR = $(wildcard $(RIOTCPU)/$(CPU)/include/vendor)
```

Comment:
I'd prefer an immediate assignment here, instead of a lazy one:
```make
CPUVENDOR := $(wildcard $(RIOTCPU)/$(CPU)/include/vendor)
```

Reply:
yeah, should work here.